portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,29 +27,45 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
+//An object that will be converted to JSON. Contains comments and their sentiment scores
+class CommentSentiment {
+    ArrayList<String> comments;
+    ArrayList<Double> scores;
+
+    CommentSentiment(ArrayList<String> c, ArrayList<Double> s) {
+        this.comments = c;
```

Comment:
it looks like you pass in entirely newly constructed lists - you could also initialize these lists in the constructor so you don't need to define them outside of this class.

Reply:
Thanks, should be fixed now!